packages/phone-number-privacy/combiner/package.json

```diff
@@ -8,16 +8,19 @@
   "scripts": {
     "dev": "yarn build && firebase serve --only functions",
     "deploy:staging": "yarn build && firebase deploy --only functions --project celo-phone-number-privacy-stg",
-    "deploy:prod": "yarn build && firebase deploy --only functions --project celo-phone-number-privacy",
+    "deploy:alfajores": "yarn build && firebase deploy --only functions --project celo-phone-number-privacy",
+    "deploy:prod": "yarn build && firebase deploy --only functions --project celo-pgpnp-mainnet",
     "config:get:staging": "firebase functions:config:get --project celo-phone-number-privacy-stg",
-    "config:get:prod": "firebase functions:config:get --project celo-phone-number-privacy",
+    "config:get:alfajores": "firebase functions:config:get --project celo-phone-number-privacy",
```

Comment:
Getting crowded here haha. It's np but if we have to add another lets make a shared script instead :)

Reply:
haha ya should probably just make a map of env --> project name...